<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lansetwindow</class>
 <widget class="QWidget" name="lansetwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/tu/Res/btn_nosel.png);</string>
  </property>
  <widget class="QWidget" name="m_pWdg_Panel_LanSet" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="m_pBtn_SimpleCh">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>300</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;border:none;background:transparent;font-size: 35px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtn_TraditionCh">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>300</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;border:none;background:transparent;font-size: 35px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtn_English">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>300</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;border:none;background:transparent;font-size: 35px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtn_French">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>300</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;border:none;background:transparent;font-size: 35px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtn_German">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>300</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;border:none;background:transparent;font-size: 35px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Return">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>690</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/Set/return.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
